apps/webpack.js

```diff
@@ -38,7 +38,6 @@ var baseConfig = {
         loader: "babel",
         query: {
           cacheDirectory: path.resolve(__dirname, '.babel-cache'),
-          sourceMaps: true,
```

Comment:
no. The docs for the babel-loader say that this option is actually ignored now. It just looks at the devtool option in the  main config.